dev-docs/source/MVPTutorial/MatPlotlib.rst

```diff
@@ -0,0 +1,73 @@
+==================
```

Comment:
Nearly there. The `P` -> `p` here.

For the rest of this document please use matplotlib in all lower case.